orientdb/pom.xml

```diff
@@ -63,21 +63,25 @@
 
         <dependency>
             <groupId>com.orientechnologies</groupId>
-            <artifactId>orientdb-core</artifactId>
+            <artifactId>orientdb-core</artifactId>            
         </dependency>
 
         <dependency>
             <groupId>com.orientechnologies</groupId>
             <artifactId>orientdb-jdbc</artifactId>
         </dependency>
-
-        <!-- Testing -->
-        <!-- //TODO Remove this -->
+        <dependency>
+            <groupId>org.parboiled</groupId>
+            <artifactId>parboiled-java</artifactId>
+        </dependency>
         <dependency>
             <groupId>org.hibernate</groupId>
-            <artifactId>hibernate-search-orm</artifactId>
-            <scope>test</scope>
+            <artifactId>hibernate-search-engine</artifactId>
+            <type>jar</type>
```

Comment:
I added hibernate-search-engine and hibernate-search-orm  besause  some tests with using JPQL queris,  uses classes from these artifacts